packages/react-app/src/services/bounty.service.ts

```diff
@@ -12,14 +12,19 @@ export const getFilters = (query: NextApiQuery): FilterParams => {
 	 * Retrieve implemented filters from the query string params
 	 */
 	const filters = {} as FilterParams;
-	
+
 	if (typeof query.status === 'string') filters.status = query.status;
 	if (typeof query.search === 'string') filters.search = query.search;
 	if (typeof query.customerId === 'string') filters.customerId = query.customerId;
+	if (typeof query.userId === 'string') filters.userId = query.userId;
 	
 	if(query.lte) filters.$lte = Number(query.lte);
 	if(query.gte) filters.$gte = Number(query.gte);
 
+	if (query.createdByMe) filters.createdByMe = Boolean(JSON.parse(query.createdByMe as string));
```

Comment:
Query params in Next are of type `string | string[]`, because you can do:

`?createdByMe=true&createdByMe=false`

Which resolves to:

```ts
query: { createdByMe: ['true', 'false'] }
```
So if I passed `createdByMe=false&createdByMe=false` I think that might resolve to `true` in the above example

Reply:
It was actually working correctly. But the logic has been changed, so it's moot